src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="POST" 
+      autocomplete="off">
```

Comment:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="POST" 
      autocomplete="off">
```
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="POST" 
      autocomplete="off"
    >
```
Tag’s closing bracket should be on the same level as opening one.